mysql-test/suite/innodb/r/check_ibd_filesize,32k.rdiff

```diff
@@ -1,5 +1,3 @@
---- mysql-test/suite/innodb/r/check_ibd_filesize.result	2022-08-16 17:28:06.462350465 +0530
-+++ mysql-test/suite/innodb/r/check_ibd_filesize.reject	2022-08-16 17:29:25.129637040 +0530
```

Comment:
You should retain these lines, but only omit the part that starts with a timestamp. Example:
```diff
--- check_ibd_filesize.result
+++ check_ibd_filesize.result
```
If you omit the lines, then there will be additional junk in the output on the Windows builder, which uses a suitably old version of the `patch` utility.